scripts/build.ps1

```diff
@@ -1273,10 +1219,8 @@ function Build-SpecificProjects {
     # FrameworksAndOutDirs format ("<target_framework>", "<output_dir>").
     $FrameworksAndOutDirs = (
         ("net462", "net462\win7-x64"),
-        ("netstandard1.0", "netstandard1.0"),
-        ("netstandard1.3", "netstandard1.3"),
         # REVIEW ME: Why do we copy netstandard2.0 into netcorecorapp2.1?
-        ("netstandard2.0", "netcoreapp2.1"),
+        ("netstandard2.0", "netstandard2.0"),
```

Comment:
I have fixed this copy folder. If there was a good reason to copy netstandard dlls into netcoreapp2.1 folder, please let me know. I will revert and add a comment.